tap_appsflyer/exceptions.py

```diff
@@ -0,0 +1,128 @@
+class appsflyerError(Exception):
+    """class representing Generic Http error."""
+
+    def __init__(self, message=None, response=None):
+        super().__init__(message)
+        self.message = message
+        self.response = response
+
+
+class appsflyerBackoffError(appsflyerError):
+    """class representing backoff error handling."""
```

Comment:
First letter in doc-string should be camelcase. Please make this change throughout this file.

```suggestion
    """Class representing backoff error handling."""
```